app/models/organization.rb

```diff
@@ -0,0 +1,6 @@
+class Organization < ActiveRecord::Base
```

Comment:
I can do that, Rails wouldn't let me create a model with the same name as the app. I'll update it to `GithubClassroom`, but I'm open to suggestions for other names.